app/controllers/comments_controller.rb

```diff
@@ -2,6 +2,7 @@ class CommentsController < ApplicationController
   before_action :logged_in_user, only: [:create, :destroy, :show]
   before_action :set_event
   before_action :correct_user, only: :destroy
+  before_action :authenticate_user!, except: :show
```

Comment:
các bạn đang dùng những built-in method được devise hỗ trợ, như vậy method `logged_in_user` có cần dùng nựa ko nhỉ? 8->